api/src/opentrons/protocol_engine/state/commands.py

```diff
@@ -41,23 +41,23 @@ def __init__(self, state: CommandState) -> None:
         """Initialize the view of command state with its underlying data."""
         self._state = state
 
-    def get_command_by_id(self, command_id: str) -> Command:
+    def get(self, command_id: str) -> Command:
         """Get a command by its unique identifier."""
         try:
             return self._state.commands_by_id[command_id]
         except KeyError:
             raise CommandDoesNotExistError(f"Command {command_id} does not exist")
 
-    def get_all_commands(self) -> List[Tuple[str, Command]]:
+    def get_all(self) -> List[Command]:
         """Get a list of all commands in state, paired with their respective IDs.
```

Comment:
```suggestion
        """Get a list of all commands in state.
```
Docstring update to match change in type signature.